demo/lib/vm.bash

```diff
@@ -630,9 +630,11 @@ vm-force-restart() { # script API
     # and fall back to vm-reboot instead.
     if vm-is-govm; then
         vm-command "shutdown -h now"
```

Comment:
Shouldn't this be 'shutdown -r now' ? Otherwise I'd expect the ssh wait test to always fail and the reboot happen by reset.